Fixed iostat manpage link

sebgod opened the pull request:
The referenced link http://linuxcommand.org/man_pages/iostat1.html for iostat failed with a 404.

azurecla commented:
Hi __@matt1883__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mahi). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@matt1883 : Thanks for your contribution to the Azure documentation! The author, @nitinme, has been notified to review your proposed change.

nitinme commented:
@matt1883, thanks for the edits. I hope you have already tested the code in the article after these edits.

nitinme commented:
@cjgronlund Please merge this PR.

nitinme commented:
@cjgronlund Please merge this PR.

sebgod committed:
Fixed iostat manpage link

The referenced link http://linuxcommand.org/man_pages/iostat1.html for iostat failed with a 404.
